js/script.js

```diff
@@ -0,0 +1,20 @@
+const headings = document.querySelectorAll( 'h2, h3, h4' );
```

Comment:
Ehhh....this seems dangerous to me to have it be so global. My inclination would be to add a body class similar to c4aa-duotone and the grid and apply to headings within `.c4aa-clipPaths` - that way the file naming matches up as well. What do you think @zoracreates?

Reply:
Going to merge and we can update if it becomes an issue!!

Reply:
Sounds good. Thanks! I didn't get a chance to work on this earlier.